### PR TITLE
blogs are now editable

### PR DESCRIPTION
**Pull request by NSPatton**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
Please describe the tests required to verify your changes. Provide instructions so PR Tester can check functionality. Please also list any relevant details for your tests
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [ ] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by cbjones93** [comment]
Not working for me, crashes whenever trying to make changes.

File: TabloidCLI/Repositories/BlogRepository.cs [hunk 1]
```diff
@@ -73,9 +73,26 @@ OUTPUT INSERTED.id
...
         {
-            throw new NotImplementedException();
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"UPDATE blog
+                                        SET Title = @title,
+                                            URL = @url2
```
Comment (bvinson23) [inline TabloidCLI/Repositories/BlogRepository.cs:85]: I think @url2 needs to be @url. It's giving me an error at this point.

**Review by bvinson23 — CHANGES_REQUESTED** [review]
See line 85 of BlogRepository for my comment.

**Review by bvinson23 — APPROVED** [review]
All good now!

**Pull request merged**